uamqp/async_ops/client_async.py

```diff
@@ -474,7 +467,7 @@ def __init__(
             error_policy=None,
             keep_alive_interval=None,
             **kwargs):
-        self.loop = loop or get_running_loop()
```

Comment:
same comment as before, should we keep this (or tweaked a bit) to keep the `get_running_loop()` behavior for python3.6 -> python3.9

Reply:
I think removing it should be fine since users should not be resetting this anyway. Made it a read-only property.